### DISCUSSION TITLE
Converting PDF to TIFF using Imagemagick, making portion of the page turning into black

### DISCUSSION THREAD

**Discussion by vikramhw** [ImageMagick/ImageMagick#3890] [Help]
While trying to convert PDF to TIF Multipage using Imagemagick, certain portion of the page  turning into black.

Version: ImageMagick 7.1.0-0 Q16 x86_64 2021-06-13 https://imagemagick.org
Copyright: (C) 1999-2021 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI OpenMP(3.1)
Delegates (built-in): bzlib djvu fontconfig freetype jng jpeg lzma openexr pangocairo png tiff x xml zlib

Sample file: https://www.dropbox.com/s/htfe177j2rnq0wo/Sample1.pdf?dl=0

Converted file screenshot: https://www.dropbox.com/s/gb7wkumj1eitvsu/ConvertedPageScreenshot.jpg?dl=0

Command used: /usr/local/bin/magick convert -debug "Blob" -density 600X600 Sample1.pdf -type Grayscale -compress Group4 -monochrome -quality 100 -depth 1 -units PixelsPerInch -define connected-components:area-threshold=3 -define connected-components:mean-color=true -define connected-components:8 Sample1.tif

Please suggest how can we get proper converted image.

**Comment by snibgo** [comment]
Your PDF is mostly transparent, ie transparent black. Your commands remove transparency, which leaves black.

I suggest after reading the PDF, you flatten against a white background, or whatever colour you want.

As you use v7, I suggest using plain "magick", not "magick convert".

```
magick -debug "Blob" -density 600X600 Sample1.pdf -background White -layers Flatten -type Grayscale {... and so on}
```

**Reply by vikramhw** [reply]
Thanks snibgo for the response.

It removed the black portion of the page but there's a side effect. Grey portion in the logo and grey portion in text also turning into white.

white Image : https://www.dropbox.com/s/ao4wzz8yab14s0g/2021-07-01_19-49-46.jpg?dl=0

magick -debug "Blob" -density 300X300 sample.pdf -background White -layers Flatten -type Grayscale -compress Group4 -quality 100 -depth 1 -units PixelsPerInch  Sample.tif

**Comment by snibgo** [comment]
"-compress Group4" reduces the image to black and white only. If you want to retain grays, don't use that compression.

Or you could remap to black and white, with dithering, before the compression. Then your gray shades will come out as black and white dots.
